cookbooks/muparser_temperature_example/muparser_temp_ex.prm

```diff
@@ -0,0 +1,168 @@
+# For the "simple" model this will be constant
+subsection Formulation
+  set Formulation = custom
+  set Mass conservation = incompressible
+  set Temperature equation = reference density profile
+end
+
+# GEOMETRY AND MODEL SIZE
+# Want to 2D box 5000 km wide by 1000 km deep
+# In METERS  5e6 x 1e6 
+# repetitions divides the whole domain into 5 boxes (1000 x 1000 km)
+# as the 0th level mesh refinement: this is needed so elements are squares
+# and not elongated rectangles.
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 5e6
+    set Y extent = 1e6
+    set X repetitions = 5
+    set Y repetitions = 1
+  end
+end
+
+# MESH INFORMATION
+# Refine the upper 150 km of the mesh so lithosphere structure is resolved.
+# For depth of 1000km, level =7  gives element size of 1000/(2^7) = 7.8125 km
+# Total refinements = initial global + intial adaptive = 7
+# Function expression asks: if ((ymax-y<=LITH) then refinement level = 7, else
+# refinement level = 0
+subsection Mesh refinement
+  set Initial global refinement                = 3
+  set Minimum refinement level		       = 3
+  set Initial adaptive refinement              = 4
+  set Time steps between mesh refinement       = 1
+  set Strategy 					= minimum refinement function
+  subsection Minimum refinement function
+    set Coordinate system   = cartesian
+    set Variable names      = x,y
+    set Function constants = YMAX=1e6,LITH=1.5e5
+    set Function expression = ((YMAX-y<=LITH) ? 7: 0)
+  end
+end
+
+# INITIAL TEMPERATURE  
+# Half-space cooling model increasing with age from x>0 to xtrm
+# For x>xtrm, half-space cooling model with a fixed age.
+# illustrates nested if-else syntax using lazy-expression (? : )
+# Lazy-expression is useful because it only evaluates the expression for
+# the "true" condition (which can avoid things like unwanted divide by zero).
+subsection Initial temperature model
+   set Model name = function
+   subsection Function
+      set Variable names = x,y
+      set Function constants = ymax=1.0e6, xtrm=2.200e6, vsub=7.927e-10, \
+			       ageop=9.46e14, Tm=1673, Ts=273, kappa=1e-6 
+      set Function expression = (((x>0.0) && (x<=xtrm)) ? \
+             (Ts + (Tm-Ts)*(1-erfc((ymax-y)/(2*sqrt(kappa*(x/vsub)))))) : \
+	  (x>xtrm) ? (Ts + (Tm-Ts)*(1-erfc((ymax-y)/(2*sqrt(kappa*ageop))))) :\
+             (Tm) ) 
+   end
+end 
+ 
+# TEMPERATURE/FLUX BOUNDARY CONDITIONS
+# Set boundary types and values for temperature
+# Default is zero-flux, so here need to set top and bottom to fixed temperature
+# Top T=273 K, Bottom T=1673 K
+# Left Flux = 0, Right Flux = 0  (even though this say "Left temperature")
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = bottom, top
+  set List of model names = box
+
+  subsection Box
+    set Bottom temperature = 1673
+    set Left temperature   = 0
+    set Right temperature  = 0
```

Comment:
To avoid the comment about left and right flux above, you can simply remove the left temperature and right temperature parameters here. The numbers here do not matter at all, because you removed the `left` and `right` boundaries from the parameter `Fixed temperature boundary indicators` above (this is what makes them zero-flux boundaries).